### ISSUE TITLE
Error: Uncaught (in promise): TypeError: Cannot read property 'updated_at' of undefinedTypeError: Cannot read

### ISSUE THREAD

**Issue by aleliga** [johannesjo/super-productivity#1414]
### Steps to Reproduce

1.
2.
3.
4.

### Stacktrace
```
Error: Uncaught (in promise): TypeError: Cannot read property 'updated_at' of undefined
TypeError: Cannot read property 'updated_at' of undefined
    at t.<anonymous> (file:///opt/superProductivity/resources/app.asar/dist/main.10cd9b5fc9f506c72dc8.js:1:2651972)
    at Generator.next (<anonymous>)
    at o (file:///opt/superProductivity/resources/app.asar/dist/main.10cd9b5fc9f506c72dc8.js:1:2649733)
    at l.invoke (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:6534)
    at Object.onInvoke (file:///opt/superProductivity/resources/app.asar/dist/main.10cd9b5fc9f506c72dc8.js:1:140110)
    at l.invoke (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:6474)
    at t.run (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:1933)
    at file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:16742
    at l.invokeTask (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:7152)
    at Object.onInvokeTask (file:///opt/superProductivity/resources/app.asar/dist/main.10cd9b5fc9f506c72dc8.js:1:139926)
    at Z (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:15947)
    at file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:15036
    at a (file:///opt/superProductivity/resources/app.asar/dist/main.10cd9b5fc9f506c72dc8.js:1:2662059)
    at l.invoke (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:6534)
    at Object.onInvoke (file:///opt/superProductivity/resources/app.asar/dist/main.10cd9b5fc9f506c72dc8.js:1:140110)
    at l.invoke (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:6474)
    at t.run (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:1933)
    at file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:16742
    at l.invokeTask (file:///opt/superProductivity/resources/app.asar/dist/polyfills.3f21005097f2d4ba8d4e.js:1:7152)
    at Object.onInvokeTask (file:///opt/superProductivity/resources/app.asar/dist/main.10cd9b5fc9f506c72dc8.js:1:139926)
```

### Meta Info
META: SP7.3.3 Electron – it – Linux x86_64 – Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) superProductivity/7.3.3 Chrome/91.0.4472.124 Electron/13.1.6 Safari/537.36

### Actions Before Error
```
1627719703352: [Persistence] Save to DB 
1627719703812: [Task] Toggle start 
1627719703815: [Persistence] Save to DB 
1627719703816: [Task] UnsetCurrentTask 
1627719711155: [Task] SetSelectedTask 
1627719715154: [Task] Update Task 
1627719715157: [Persistence] Save to DB 
1627719716138: [Task] SetSelectedTask 
1627719723253: [Task] Update Task 
1627719723270: [Persistence] Save to DB 
1627719727348: [Task] SetSelectedTask 
1627719730337: [Task] Move to archive 
1627719730357: [Persistence] Save to DB 
1627719730358: [Persistence] Save to DB 
1627719730358: [Persistence] Save to DB 
1627719730358: [Task] UnsetCurrentTask 
1627719730430: [Persistence] Save to DB 
1627719737169: [WorkContext] Set Active Work Context 
1627719737223: [Task] SetSelectedTask 
1627719737470: [Bookmark] Load Bookmark State 
1627719737475: [Note] Load Note State 
1627719737475: [Project] Load Project related Data Success 
1627719737476: [Improvement] Clear Hidden Improvements 
1627719737476: [Improvement] Delete multiple Improvements 
1627719742404: [Task] SetSelectedTask 
1627719744574: [Task] Move to archive 
1627719744583: [Persistence] Save to DB 
1627719744583: [Persistence] Save to DB 
1627719744583: [Task] UnsetCurrentTask 
1627719744618: [Persistence] Save to DB
```

**Comment by johannesjo** [comment]
Hi there! Thank you very much for reporting. Could you maybe provide a little bit more input?

**Comment by ishanarora** [comment]
Duplicate of #1634

**Comment by ishanarora** [comment]
Fixed in 48e2b919f

**Comment by johannesjo** [comment]
I will try to make a new release next week.
